cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/packages.proto

```diff
@@ -274,17 +259,11 @@ message AvailablePackageSummary {
   // The name of the available package
   string name = 2;
 
-  // Latest available package version
+  // Latest available version
   //
   // The latest version available for this package. Often expected when viewing
   // a summary of many available packages.
-  string latest_pkg_version = 3;
-
-  // Latest available package app version
-  //
-  // The app version of the latest version available for this package. Often expected
-  // when viewing a summary of many available packages.
-  string latest_app_version = 4;
+  PackageAppVersion latest_version = 3;
```

Comment:
looks like you skipped number 4. here. probably just a typo

Reply:
I just didn't want to bloat the diff (was going to re-order in a separate PR, but happy to do so here). Let me do that.